src/pages/2021/rfp.mdx

```diff
@@ -4,37 +4,29 @@ import { TopicSquare } from "../../components/rfp/TopicSquare";
 
 <SiteLayout title="Request for Proposals">
 
-<div className="wrapper relative mb-8">
-  <div className="content">
-    <h1>Request for Proposals</h1>
-    <CTAButton onClick={"https://form.jotform.com/211237107190142"}>
-      Submit a Proposal
-    </CTAButton>
-  </div>
-  <style jsx>{`
-    .wrapper {
-      background-image: url("/images/pattern/rainbox-box-pattern-desaturated.png");
-      background-repeat: repeat;
-      background-size: contain;
-      min-height: 20rem;
-      display: flex;
-      align-items: center;
-      justify-content: center;
-      padding: 2rem;
-      font-size: 2rem;
-      text-align: center;
-      font-weight: bold;
-    }
-  `}</style>
+<div className="border-b-4 border-mulberry border-solid pb-4 mb-4 text-center">
+    <p className="text-5xl font-display font-black">Requests for Proposals</p>
 </div>
 
+
 Out in Science, Technology, Engineering, and Mathematics (oSTEM) is now
 accepting program proposals for the virtual 11th Annual oSTEM Conference 2021.
 For more information on why we decided to go virtual, read the full statement.
 
 Successful programs will touch on oSTEM’s mission statement and also incorporate
 our core values: integrity, intentionality, respect, passion, and grit.
 
+<div className="bg-tint-spruce-light text-2xl mt-4 p-8 rounded">
+    <p className="flex items-center">
+        <svg xmlns="http://www.w3.org/2000/svg" className="h-6 w-6 mr-3" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+            <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M12 8v4m0 4h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z" />
+        </svg>
+        Our Call for Proposals is currently closed.
```

Comment:
'Our Request for Proposals is currently closed.'

Need to keep language consistent within a page pls.